man/man5/zfs-module-parameters.5

```diff
@@ -2165,6 +2164,41 @@ See the section "ZFS I/O SCHEDULER".
 Default value: \fB1\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_vdev_nia_delay\fR (int)
+.ad
+.RS 12n
+To reduce the effects of non-interactive I/O (scrub, resilver, removal,
+initialize and rebuild) on interactive I/O (both sync and async) latency
+they are limited to *_min_active while there are outstanding interactive
+I/Os, or until an additional zfs_vdev_nia_delay I/O's complete after the
+last interactive I/O.
```

Comment:
I found this a little hard to understand.  How about something like:

`For background-priority I/O (scrub, resilver, removal, initialize, and rebuild), the number of concurrently-active I/O's is limited to *_min_active, unless the system is "idle".  When there are no foreground-priority I/Os active (sync or async), and zfs_vdev_nia_delay background I/Os have completed since the last foreground I/O, then the system is considered to be "idle", and the number of concurrently-active background-priority I/O's is increased to *_max_active.`